hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/test/java/org/apache/hadoop/yarn/server/federation/store/impl/FederationStateStoreBaseTest.java

```diff
@@ -905,7 +905,7 @@ public void testGetMasterKeyByDelegationKey() throws YarnException, IOException
   }
 
   @Test
-  public void testRemoveStoredMasterKey() throws YarnException, IOException {
+  public void testRemoveStoredMasterKey() throws IOException, YarnException {
```

Comment:
Avoid?

Reply:
Thank you so much for reviewing the code, I will fix it.